scripts/install_sshnp

```diff
@@ -409,6 +409,19 @@ do_update() {
   echo; echo "Update complete!";
 }
 
+deprecation_msg() {
+  case "$SSHNP_VERSION" in
+    # TODO remove latest to signify deprecation on the launch of v4.0.0
+    latest|1.*|2.*|3.*|4.0.0-test.*|4.0.0-rc.*)
+      # no message, this group is expected to use this script
```

Comment:
Added an automated deprecation message which will automatically trigger on 4.0.0 and later, we just have to remove latest from the safe-list once we release 4.0.0